### PR TITLE
UX design vocabulary-based changes

### PR DESCRIPTION
**Pull request by paulfioravanti**
This PR proposes to add some entries that I came across while finishing the [UX design vocabulary lesson](https://didoesdigital.com/typey-type/lessons/collections/user-experience/ux-vocabulary/) on Typey Type.

Since the [`ux-design.json`](https://github.com/didoesdigital/steno-dictionaries/blob/master/dictionaries/ux-design.json) dictionary is a personal dictionary, I have left it alone and instead primarily focused on trying to create condensed strokes for words that were not already in Plover.

Each individual change is in a separate commit, so I'm happy to make any targeted changes.

### CODE REVIEW

File: dictionaries/condensed-strokes.json [hunk 1]
```diff
@@ -237,6 +240,7 @@
 "EPB/TKPWROS/-G": "engrossing",
 "EPB/TRAOET/KWREUS": "entreaties",
 "EPL/PWAES/-S": "embassies",
+"EPL/S-PTKARB": "em dash",
```
Comment (didoesdigital) [inline dictionaries/condensed-strokes.json:243]: `S-PTKARB` doesn't appear to be in steno order. Are you missing a `/` @paulfioravanti ?

File: dictionaries/dict.json [hunk 2]
```diff
@@ -1101,6 +1101,7 @@
 "A*PB/A*PB/AR/PWOR": "Ann Arbor",
 "A*PB/A*PB/ARB/O*R": "Ann Arbor",
 "A*PB/A*PB/HAEURB": "Anne Heche",
+"A*PB/HREUBGS": "Analytics",
```
Comment (didoesdigital) [inline dictionaries/dict.json:1104]: I think `A*PB` is a suffix, not a capitalised prefix: `"A*PB": "{^an}",` — am I missing something?

Comment (didoesdigital) [reply dictionaries/dict.json:1104]: Oh, I worked it out — new entry in Plover

**Comment by didoesdigital** [comment]
Thanks @paulfioravanti ! One small comment, otherwise it looks good!

I can see how adding these to `condensed-strokes.json` makes sense, although you're also welcome to make suggestions for my personal dictionaries.

File: dictionaries/condensed-strokes.json (hunk 1 above)
Comment (paulfioravanti) [reply dictionaries/condensed-strokes.json]: Thanks for that, @didoesdigital. Definitely a typo: should be `EPL/S-P/TKARB`. Will fix.

**Comment by didoesdigital** [comment]
Thanks @paulfioravanti . I made one small subsequent commit: https://github.com/didoesdigital/steno-dictionaries/commit/b40a979ba15a3e52bc8b5f064ff8c98e5f6cfa53

**Pull request merged**